src/goto-harness/recursive_initialization.h

```diff
@@ -35,6 +38,15 @@ struct recursive_initialization_configt
   std::set<irep_idt> pointers_to_treat_as_cstrings;
 
   std::string to_string() const; // for debugging purposes
+
+  /// Parse the options specific for recursive initialisation
+  /// \param option: the user option name
+  /// \param values: the (one-or-more) values for this option
+  /// \return true if the option belonged to recursive initialisation and was
+  /// successfully parsed here
```

Comment:
Nit pick: indent comment that spans multiple lines (as documented in CODING_STANDARD.md)

Reply:
Done.